dozer/cogs/info.py

```diff
@@ -1,7 +1,7 @@
 """Provides commands for pulling certain information."""
 import math
 import typing
-from datetime import timezone, datetime, date
```

Comment:
The changes in this file seem unrelated to this PR

Reply:
It's in the "as well as fixing..." section, specifically related to how I vs others import the datetime library. Seemed easier to me to just add timedelta and fix my lines than just import datetime and run everything off that.
There's certain commands (including &stats) that only work on dozer2.0 right now due to the incorrect import